packages/web-components/src/radio-group/radio-group.stories.ts

```diff
@@ -0,0 +1,162 @@
+    <fluent-radio value="pear"> Pear </fluent-radio>
+    <fluent-radio value="banana"> Banana </fluent-radio>
+    <fluent-radio value="orange"> Orange </fluent-radio>
+  </fluent-radio-group>
+`;
+
+export default {
+  title: 'Components/RadioGroup',
+  args: {
+    disabled: false,
+    orientation: 'horizontal',
+  },
+  argTypes: {
+    checked: {
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        type: {
+          summary: 'Sets checked state on radio',
+        },
+        defaultValue: {
+          summary: 'false',
+        },
+      },
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        type: {
+          summary: 'Sets default state on radio',
+        },
+        defaultValue: {
+          summary: 'false',
+        },
+      },
+    },
+    stacked: {
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        type: {
+          summary: 'Sets layout of radio buttons',
+        },
+        defaultValue: {
+          summary: 'false',
+        },
+      },
+    },
+    orientation: {
+      control: {
+        type: 'select',
+        options: Object.values(RadioGroupOrientation),
+      },
+      defaultValue: 'primary',
+      table: {
+        type: {
+          summary: 'Sets orientation of radio group',
+        },
+        defaultValue: {
+          summary: 'horizontal',
+        },
+      },
+    },
+  },
+} as RadioGroupStoryMeta;
+
+export const RadioGroup = renderComponent(storyTemplate).bind({});
+
+export const RadioGroupLabelledby = renderComponent(html<RadioGroupStoryArgs>`
+  <fluent-radio-group aria-labelledby="label-2" name="radio-story">
+    <span id="label-2" slot="label">Favorite Fruit</span>
+    <fluent-radio value="apple"> Apple </fluent-radio>
+    <fluent-radio value="pear"> Pear </fluent-radio>
+    <fluent-radio value="banana"> Banana </fluent-radio>
+    <fluent-radio value="orange"> Orange </fluent-radio>
+  </fluent-radio-group>
+`);
+
+export const RadioGroupLayoutVertical = renderComponent(html<RadioGroupStoryArgs>`
+  <fluent-radio-group aria-labelledby="label-3" orientation="vertical" name="radio-story">
+    <span id="label-3" slot="label">Favorite Fruit</span>
+    <fluent-radio value="apple"> Apple </fluent-radio>
+    <fluent-radio value="pear"> Pear </fluent-radio>
+    <fluent-radio value="banana"> Banana </fluent-radio>
+    <fluent-radio value="orange"> Orange </fluent-radio>
+  </fluent-radio-group>
+`);
+
+export const RadioGroupLayoutHorizontal = renderComponent(html<RadioGroupStoryArgs>`
+  <fluent-radio-group aria-labelledby="label-4" orientation="horizontal" name="radio-story">
+    <span id="label-4" slot="label">Favorite Fruit</span>
+    <fluent-radio value="apple"> Apple </fluent-radio>
+    <fluent-radio value="pear"> Pear </fluent-radio>
+    <fluent-radio value="banana"> Banana </fluent-radio>
+    <fluent-radio value="orange"> Orange </fluent-radio>
```

Comment:
This is an absolute nit, but the whitespace here might actually impact the visuals of the controls. I'm not sure if it is, but we should probably avoid it.